index.html

```diff
@@ -4152,6 +4152,105 @@ <h2>Presenting Time Values</h2>
 
     </section>
 
+    <section>
+      <h2>Internationalization Considerations</h2>
+      <p>
+The following section describes internationalization considerations that
+developers implementing or using this specification are urged to consider in
+order to ensure that their software and documents are usable by people that
+read, write, or speak a variety of different languages. As a general rule, this
+specification is used by system software to secure digital information and
+the internationalization support of that digital information is a function of
+the data that is being secured, and not this specification.
+      </p>
+
+      <section>
+        <h2>Internationalization in JSON</h2>
+
+        <p>
+There is currently no standard way to express language and text direction
```

Comment:
```suggestion
There is currently no standard way to express language and/or text direction
```

Reply:
This section was removed as requested in https://github.com/w3c/vc-data-integrity/pull/213#discussion_r1364183506, so the change suggestion wasn't applied.